core/dbt/rpc.py

```diff
@@ -267,12 +267,28 @@ def handle(self, kwargs):
         self.started = time.time()
         self.timeout = kwargs.pop('timeout', None)
         self.queue = multiprocessing.Queue()
-        self.process = multiprocessing.Process(
-            target=_task_bootstrap,
-            args=(self.task, self.queue, kwargs)
-        )
-        self.process.start()
-        result = self.get_result()
+        if self.task.args.single_threaded:
```

Comment:
what's up with this change?

Reply:
Oh, I was using this during debugging and it was broken for `dbt --single-threaded rpc` so I fixed it. Nothing to see here!